src/MyEventEmitter.js

```diff
@@ -1,14 +1,63 @@
 'use strict';
 
 class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this.listeners = {};
+  }
+  on(event, callback) {
+    if (!this.listeners[event]) {
+      this.listeners[event] = [];
+    }
+    this.listeners[event].push(callback);
+  }
+  once(event, callback) {
+    const wrapper = (...args) => {
+      callback(...args);
+      this.off(event, wrapper);
+    };
+
+    this.on(event, wrapper);
+  }
+  off(event, callback) {
+    if (this.listeners[event]) {
+      this.listeners[event] = this.listeners[event].filter(
+        (listener) => listener !== callback,
```

Comment:
There is a trailing comma after the callback comparison in the filter function. This is not necessary and might cause issues in environments that do not support trailing commas in function parameter lists. Consider removing it.